packages/tools/src/tools/annotation/ArrowAnnotateTool.ts

```diff
@@ -776,6 +776,21 @@ class ArrowAnnotateTool extends AnnotationTool {
         continue;
       }
 
+      const options = this.getLinkedTextBoxStyle(styleSpecifier, annotation);
```

Comment:
so is this just hiding it ? or you don't want the stats to get calculated either?

Reply:
The statistics are already calculated, which is in "cachedStats" in the annotation. Just to not show the "textBox" in the viewer.